keps/sig-node/3386-kubelet-evented-pleg/README.md

```diff
@@ -348,6 +349,9 @@ We expect no non-infra related flakes in the last month as a GA graduation crite
 - Add E2E Node Conformance presubmit job in CI 
 - Add E2E Node Conformance periodic job in CI
 
+#### GA
```

Comment:
will there be "beta" enabled by default? Maybe call it beta2?

Reply:
I don't think we will enable it by default only in GA

Reply:
> will there be "beta" enabled by default? Maybe call it beta2?

Feature gates only go from Alpha->Beta->GA, https://github.com/kubernetes/kubernetes/blob/03ff890ef47a7dd0d207f09bda963d5d970a7535/staging/src/k8s.io/component-base/featuregate/feature_gate.go#L81 

I wasn't able to find any declaration for `beta2` for feature gates.

Reply:
Maybe just add a section `#Beta (Enabled by default)`?

Reply:
But we already have enhancement merged for Beta, https://github.com/kubernetes/enhancements/pull/3817

The corresponding PR - https://github.com/kubernetes/kubernetes/pull/115967

Without getting more stable signal from [recently added node e2e](https://testgrid.k8s.io/sig-node-cri-o#ci-crio-cgroupv1-evented-pleg) as well as some stress tests I am not sure if we should enable it by default.

Reply:
Oh I think now I get what you are trying to say, instead of adding `GA` keep it in `Beta` and make it `Enable by default`.  

That makes sense. Sorry for the confusion. Thanks.

Reply:
have you decided against making it a criteria for Beta (enabled by default)? 

```suggestion
#### Beta (Enabled by default)
```

Reply:
I thought @dchen1107 was [okay](https://github.com/kubernetes/kubernetes/pull/115967#issuecomment-1462684923) to delay stress test to GA. 

Anyway, I have made it `Beta (Enabled by default)` as per your comment.